2-classes-db/task.md

```diff
@@ -1,75 +1,113 @@
 # Class Database
+
 Run the following commands:
+
 ```
 createdb homework
 psql -d homework -f setup.sql
 ```
+
 This will create and populate a database called homework.
 
 It is based on real data from the government found at
 https://data.gov.uk/dataset/72eaec8e-0d32-4041-a553-87b852abee64/spend-over-25-000-in-worcestershire-acute-hospitals-nhs-trust
 
 You can work with it by running
+
 ```
 psql homework
 ```
+
 ## Submission
 
 Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands after each question.
 
 ### 1. Show the date, transaction_no, description and amount for those transactions whose amount is between £30,000 and £31,000.
+
 ```sql
+SELECT date,transaction_no,description,amount FROM spends WHERE amount >= 30000 AND amount <= 31000;
 
+7 rows
 ```
+
 ### 2. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee'.
+
 ```sql
+SELECT date,transaction_no,supplier_inv_no,description,amount FROM spends WHERE description LIKE '%fee%';
 
+(3 rows)
 ```
+
 ### 3. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'Fee'.
+
 ```sql
+SELECT date,transaction_no,supplier_inv_no,description,amount FROM spends WHERE description LIKE '%fee%';
 
+(6 rows)
 ```
+
 ### 4. Show the date, transaction_no, supplier_inv_no, description and amount for those transactions whose description includes the word 'fee' (case insensitive). You will need to search 'https://www.postgresql.org/docs/' to solve this.
+
 ```sql
+SELECT date,transaction_no,supplier_inv_no,description,amount FROM spends WHERE LOWER(description) LIKE LOWER('%fee%');
 
```

Comment:
CORRECT - This will work.  However, bear in mind that for the search string *you* are defining it, so '%fee%' is already in lower case without you having to apply the LOWER operator.  As you rightly use it, LOWER *is* necessary for use on the description field.